### ISSUE TITLE
[Bug] Getting a "TypeError: otel.setSpan is not a function" error when running hello world sample app

### ISSUE THREAD

**Issue by denjalonso** [temporalio/sdk-node#125]
**Describe the bug**
Error when running getting started (hello world sample app), step 3 "Run your Workflow"

**To Reproduce**
1. Go to https://docs.temporal.io/docs/node/getting-started/
2. Follow steps
3. Get below error when running ```node lib/worker```:
```
Worker state changed { state: 'RUNNING' }
2021-06-15T08:30:38.727Z [ERROR] Worker failed {
  error: TypeError: otel.setSpan is not a function
      at Object.instrument (/Users/dalogon/Projects/12-technoactivity-projects/22-core-services/10-mymoidapis-campaigns/temporal-test/example/node_modules/@temporalio/worker/lib/tracing.js:45:26)
      at /Users/dalogon/Projects/12-technoactivity-projects/22-core-services/10-mymoidapis-campaigns/temporal-test/example/node_modules/@temporalio/worker/lib/worker.js:488:40
      at MergeMapSubscriber.project (/Users/dalogon/Projects/12-technoactivity-projects/22-core-services/10-mymoidapis-campaigns/temporal-test/example/node_modules/@temporalio/worker/lib/worker.js:258:28)
      at MergeMapSubscriber._tryNext (/Users/dalogon/Projects/12-technoactivity-projects/22-core-services/10-mymoidapis-campaigns/temporal-test/example/node_modules/rxjs/internal/operators/mergeMap.js:67:27)
      at MergeMapSubscriber._next (/Users/dalogon/Projects/12-technoactivity-projects/22-core-services/10-mymoidapis-campaigns/temporal-test/example/node_modules/rxjs/internal/operators/mergeMap.js:57:18)
      at MergeMapSubscriber.Subscriber.next (/Users/dalogon/Projects/12-technoactivity-projects/22-core-services/10-mymoidapis-campaigns/temporal-test/example/node_modules/rxjs/internal/Subscriber.js:66:18)
      at MapSubscriber._next (/Users/dalogon/Projects/12-technoactivity-projects/22-core-services/10-mymoidapis-campaigns/temporal-test/example/node_modules/rxjs/internal/operators/map.js:55:26)
      at MapSubscriber.Subscriber.next (/Users/dalogon/Projects/12-technoactivity-projects/22-core-services/10-mymoidapis-campaigns/temporal-test/example/node_modules/rxjs/internal/Subscriber.js:66:18)
      at Observable._subscribe (/Users/dalogon/Projects/12-technoactivity-projects/22-core-services/10-mymoidapis-campaigns/temporal-test/example/node_modules/rxjs/internal/util/subscribeToArray.js:5:20)
      at Observable._trySubscribe (/Users/dalogon/Projects/12-technoactivity-projects/22-core-services/10-mymoidapis-campaigns/temporal-test/example/node_modules/rxjs/internal/Observable.js:44:25)
}
2021-06-15T08:30:38.729Z [INFO] Worker state changed { state: 'FAILED' }
```

**Expected behavior**
Starts the worker

**Screenshots/Terminal ouput**
<img width="1838" alt="Screenshot 2021-06-15 at 09 32 07" src="https://user-images.githubusercontent.com/3237551/122022010-1d0fff00-cdbe-11eb-86e0-ebd21a595c73.png">

**Versions (please complete the following information where relevant):**
 - OS: macOs Big Sur 11.4 Apple M1
 - Node v16.3.0 (tj/n)
 - ```"temporalio": "0.2.1"```
 - Temporal server with docker compose

**Additional context**
I have tried with different versions of node> = 15 also with nvm

**Comment by juanchojbarroso** [comment]
+1 here!

**Comment by bergundy** [comment]
This has already been fixed in the main branch.
Looks like opentelemetry broke semver.
I'm traveling right now but I'll cut out a patch release as soon as I'm available.

**Comment by bergundy** [comment]
Just released `temporalio@0.2.2` and `@temporalio/worker@0.4.0` which fix the issue.
We should probably monitor the installation to prevent this type of breakage in the future.
